utils/coder.js

```diff
@@ -20,6 +22,7 @@ export class Coder {
                 return code;
             }
         }
+        code = code.replace(';', '; if(bot.abort_code) return false;')
```

Comment:
Let me know if you have a better solution. The clear() function will halt the current pathfinding, but if there is a loop or multiple commands in the bot's code it will keep executing. Using this 'abort_code' variable is my fix for that.